[Intl] Add format_list filter using PHP 8.5's IntlListFormatter (#4852)
twigphp/Twig: doc/filters/format_list.rst

``format_list``

> ================

Reviewer: ```suggestion
===============
```

.. versionadded:: 3.29

>     The ``html_attr_merge`` filter was added in Twig 3.29.

Reviewer: ```suggestion
    The ``format_list`` filter was added in Twig 3.29.
```

.. note::

    The ``format_list`` filter requires the ``IntlListFormatter`` class, which is available since PHP 8.5.

> 

Reviewer: ```suggestion
```
